File: src/main/java/me/ry00001/jbot/utils/OwnerChecker.java
package me.ry00001.jbot.utils;

import java.util.stream.LongStream;
import me.ry00001.jbot.core.CommandContext;

public class OwnerChecker {
    public OwnerChecker() {}

    public boolean isOwner(CommandContext ctx) {
        return LongStream.of(ctx.bot.config.owners).anyMatch(x -> x == ctx.author.getIdLong());
    }
}

File: src/main/kotlin/me/ry00001/jbot/utils/OwnerChecker.kt
package me.ry00001.jbot.utils

import me.ry00001.jbot.core.CommandContext
import java.util.stream.LongStream

class OwnerChecker {

    fun isOwner(ctx: CommandContext): Boolean {
        return LongStream.of(*ctx.bot.config.owners!!).anyMatch { x -> x == ctx.author.idLong }
    }
}